### PR TITLE
FRW-8883 Fixed customer login so that email confirmation is checked and errors are correctly reported.

### PR DESCRIPTION
**Pull request by aleksandr-velikanov**
- Developer(s): @aleksandr-velikanov

- Ticket: https://spryker.atlassian.net/browse/FRW-8883

##### Change log

Improvements

- Updated login error messages

### CODE REVIEW

File: cypress/support/pages/yves/login/repositories/suite-login-repository.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ export class SuiteLoginRepository implements LoginRepository {
   getRegisterConfirmPasswordInput = (): Cypress.Chainable => cy.get('#registerForm_password_confirm');
   getRegisterAcceptTermsCheckbox = (): Cypress.Chainable => cy.get('#registerForm_accept_terms');
   getRegisterForm = (): Cypress.Chainable => cy.get('form[name=registerForm]');
-  getFailedAuthenticationText = (): string => 'Authentication failed';
+  getFailedAuthenticationText = (): string =>
+    'Please check that your E-mail address and password are correct and that you have confirmed your E-mail address by clicking the link in the registration message';
```
Comment (dmiseev) [inline cypress/support/pages/yves/login/repositories/suite-login-repository.ts:19]: what about `b2c-mp` and `b2b-mp` repos?

Comment (aleksandr-velikanov) [reply cypress/support/pages/yves/login/repositories/suite-login-repository.ts:19]: b2b doesn't have customer login I believe. For b2b-mp will add

Comment (dmiseev) [reply cypress/support/pages/yves/login/repositories/suite-login-repository.ts:19]: I believe that customer login functionality was implemented everywhere.

Comment (kraal-spryker) [reply cypress/support/pages/yves/login/repositories/suite-login-repository.ts:19]: if b2bmp have customer login, then b2b have for sure

**Review by kraal-spryker — APPROVED** [review]

File: cypress/support/pages/yves/login/repositories/suite-login-repository.ts (hunk 1 above)
Comment (aleksandr-velikanov) [reply cypress/support/pages/yves/login/repositories/suite-login-repository.ts:19]: Ok, updated for both

**Review by dmiseev — APPROVED** [review]

**Comment by dmiseev** [comment]
Please fix CI

**Review by dmiseev — APPROVED** [review]

**Pull request merged**